src/@types/emotion.d.ts

```diff
@@ -0,0 +1,7 @@
+import "@emotion/react";
+import { EmotionTheme } from "~/styles/emotionTheme";
+
+declare module "@emotion/react" {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  export interface Theme extends EmotionTheme {}
+}
```

Comment:
```suggestion
import "@emotion/react";
import emotionTheme from "~/styles/emotionTheme";

declare module "@emotion/react" {
  type EmotionTheme = typeof emotionTheme;
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  export interface Theme extends EmotionTheme {}
}
```

이거 EmotionTheme 타입이 d.ts에서 쓰이기 위해 존재하는 녀석 같은데 여기서 선언해서 바로 넣어주면 오케이 아닌가요?

Reply:
동규님 훨씬좋네요 ! 감사합니다 수정했습니다